beta/examples/tensorflow/classification/main.py

```diff
@@ -227,7 +227,8 @@ def run(config):
         validation_dataset,
         steps=validation_steps,
         callbacks=[get_progress_bar(
-            stateful_metrics=['loss'] + [metric.name for metric in metrics])],
+            stateful_metrics=['loss'] + [metric.name for metric in metrics]),
```

Comment:
@daniil-lyakhov when we resume the model from the checkpoint we also load the scheduler state. It looks like the root cause is that the scheduler state is not loaded correctly. Could you take a look at this?

Reply:
According to the docstring of `load_state` base method https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/schedulers.py#L211-L217
```
    def load_state(self, state: Dict[str, object]) -> None:
        """
        Loads the compression scheduler state, but does not update the state of the
        compression method.
        :param state: Output of `get_state()` method.
        """
```
If we would update the compression parameters (target loss for instance) it would break the logic of common method.

I suggest expanding purposes of this method by compression state update for tf schedulers

Reply:
Besides it should solve wrong evaluation statistics issues for magnitude sparsity and filter pruning too